source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -653,22 +654,23 @@ If you enable this policy setting, you can choose from four options for loading
  
 If you disable or do not configure this policy setting, the behavior is as if you enable this policy setting and then select option 1.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
       <DuplicateOf />
-      <Ensure>Absent</Ensure>
+      <Ensure>Present</Ensure>
       <IsNullOrEmpty>False</IsNullOrEmpty>
       <Key>HKEY_CURRENT_USER\keycupoliciesmsvbasecurity</Key>
       <LegacyId>V-99665</LegacyId>
       <OrganizationValueRequired>False</OrganizationValueRequired>
       <OrganizationValueTestString />
-      <RawString>Verify the policy value for User Configuration &gt;&gt; Administrative Templates &gt;&gt; Microsoft Office 2016 &gt;&gt; Security Settings "Load Controls in Forms3" is set to "Enabled (For a UFI or SFI signed control that supports safe and unsafe mode, load the control in unsafe mode. For an SFI signed control that only supports a safe mode configuration, load the control in safe mode. This option enforces the default configuration.)".
+      <RawString>Verify the policy value for User Configuration &gt;&gt; Administrative Templates &gt;&gt; Microsoft Office 2016 &gt;&gt; Security Settings "Load Controls in Forms3" is set to Enabled and 1 from drop down. (For a UFI or SFI signed control that supports safe and unsafe mode, load the control in unsafe mode. For an SFI signed control that only supports a safe mode configuration, load the control in safe mode. This option enforces the default configuration.)".
 
 Use the Windows Registry Editor to navigate to the following key:
 
-HKCU\keycupoliciesmsvbasecurity 
+HKCU\Software\Policies\Microsoft\VBA\Security
```

Comment:
This path may have been parsed incorrectly in the past, the same process should occur with this one as well, admx, then evaluate registry setting.

Reply:
updated hard coded rule

![porplusvba](https://user-images.githubusercontent.com/20539446/128453137-f5841318-db33-4ba7-ba4e-99ed46025f83.JPG)